### PR TITLE
GNOME 45

### LINKED ISSUE #710: Scrolling over parts of the side panel changes the calendar date
## Context
When the window is _not_ maximized, and the content in the sidebar to the left does not have enough room to be fully displayed, the entire sidebar becomes scrollable (indicated by the scrollbar that shows up with any mouse movement in that area).

## Current Behavior
If I perform a scroll action while my cursor is hovering over the calendar widget, it immediately begins flipping through the days and months.

## Desired Behavior
Scrolling the sidebar should not change the calendar date, or have any other immediate actions beyond scrolling the sidebar.

### Version, OS, etc.

Tested in Denaro 2023.9.2, from Flathub (running on Fedora 39 Developers Preview/Pre-Beta)

### LINKED ISSUE #731: GNOME 45
https://github.com/NickvisionApps/Proposals/issues/10

### PR DESCRIPTION
**Pull request by DaPigGuy**
Fixes #710 
Closes #731

### CODE REVIEW

**Comment by DaPigGuy** [comment]
Flatpak build failure caused by https://github.com/NickvisionApps/Denaro/commit/366e0a11d15153918f9d199006afc2a1ca828baa

**Comment by nlogozzo** [comment]
> Flatpak build failure likely caused by [366e0a1](https://github.com/NickvisionApps/Denaro/commit/366e0a11d15153918f9d199006afc2a1ca828baa), will look into it in the morning

Yes i know why, i will fix that ...

File: NickvisionMoney.GNOME/Blueprints/account_view.blp [hunk 1]
```diff
@@ -76,7 +89,7 @@ Adw.Bin _root {
             
             Gtk.Button _resetOverviewFilterButton {
               icon-name: "edit-select-all-symbolic";
-              tooltip-text: _("Select All Overview Filters");
+              tooltip-text: _("Select All sfjkdjdfjOverview Filters");
```
Comment (nlogozzo) [inline NickvisionMoney.GNOME/Blueprints/account_view.blp:92]: What happened here?😂

**Review by nlogozzo — CHANGES_REQUESTED** [review]

File: NickvisionMoney.GNOME/Blueprints/account_view.blp (hunk 1 above)
Comment (DaPigGuy) [reply NickvisionMoney.GNOME/Blueprints/account_view.blp]: That was a test commit :sweat_smile:

**Comment by nlogozzo** [comment]
I will finish reviewing and updating all the deps making sure everything builds once I get back home from dinner

**Pull request merged**